docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to LoopBack
+
+Contributions to Node.js include code, documentation, answering user questions, and advocating for all types of Node.js users. See our official documentation on loopback.io for more information common to all of our GitHub repositories:
+
+  - http://loopback.io/doc/en/contrib/index.html
+
+## [Principles](http://loopback.io/doc/en/contrib/Governance.html#principles)
+
+LoopBack is an open, inclusive, and tolerant community of people working together to build a world-class Node framework and tools. We value diversity of individuals and opinions, and seek to operate on consensus whenever possible. We strive to maintain a welcoming, inclusive, and harassment-free environment, regardless of the form of communication. When consensus is not achievable, we defer to the owners of each individual module; the powers of the individual owner are kept in check by the ability of the community to fork and replace dependencies on the individual module and maintainer.
+
+## [Reporting issues](http://loopback.io/doc/en/contrib/Reporting-issues.html)
+
+Issues in [strongloop/loopback-next](https://github.com/strongloop/loopback-next/issues) are the primary means by which bug reports and general discussions are made.
+
+ - [How to report an issue](http://loopback.io/doc/en/contrib/Reporting-issues.html#how-to-report-an-issue)
+ - [Disclosing security vulnerabilities](http://loopback.io/doc/en/contrib/Reporting-issues.html#security-issues)
+
+## [Contributing code](http://loopback.io/doc/en/contrib/code-contrib.html)
+
+Pull Requests are the way concrete changes are made to the code, documentation
+and tools contained in LoopBack repositories.
+
+ - [Setting up development environment](./DEVELOPING.md#setting-up-development-environment)
```

Comment:
Since most of the links are coming from `DEVELOPING.md`, I wonder if we could group them to a category sth like "what you need to know to develop LB".  It might be more convenient to the users to go to that link and read rather than going to each link and basically pointing to the same page.
Maybe it's just me...

Reply:
I actually like the separate links (even though they are all pointing to the same page).

Reply:
Here is my point of view:

 - "what you need to know to develop LB" is already grouped in `DEVELOPING.md`. Users wishing to read everything about DEVELOPING in one go can already read that file!
 - I could shorten this list and replace all items pointing to different sections of DEVELOPING with a single link pointing to that file, but I think that's worse user experience because users cannot quickly jump from CONTRIBUTING to a relevant section in DEVELOPING.

I see two kinds of users:

 - Jane would like to learn everything about developing loopback-next in one go. She can open DEVELOPING file and read through it.
 - Joe prefers to jump to the code and read only as much as is needed. Links in CONTRIBUTING makes it easier for him to see what different topics are available and choose to read only what's relevant for him.

FWIW, Node.js uses similar approach in their [CONTRIBUTING.md](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md)